documentation/writing-policies-configmap-reference.md

```diff
@@ -10,41 +10,84 @@ The Configmap Reference allows the reference of configmap values inside kyverno
 
 To refer Configmap inside any Rule provide the context inside each rule defining the list of configmaps which will be referenced in that Rule.
 
-```
+````yaml
   rules:
-    - name: add-sidecar-pod
+    - name: example-configmap-lookup
       # added context to define the configmap information which will be referred 
       context:
       # unique name to identify configmap
-      - name: mycmapRef
+      - name: dictionary
         configMap: 
           # configmap name - name of the configmap which will be referred
           name: mycmap
           # configmap namepsace - namespace of the configmap which will be referred
-```
+          namespace: test
+````
 
 Referenced Configmap Definition
 
-```
+````yaml
 apiVersion: v1
 data:
-  env: production, sandbox, staging
+  env: production
 kind: ConfigMap
 metadata:
   name: mycmap
-```
+````
 
 # Referring Value
 
 The configmaps that are defined inside rule context can be referred using the unique name that is used to identify configmap inside context.
```

Comment:
A ConfigMap that is defined inside the rule context can be referred to using its unique name within the context.